espnet2/tasks/mt.py

```diff
@@ -53,6 +59,15 @@
     type_check=AbsFrontend,
     default="embed",
 )
+specaug_choices = ClassChoices(
+    name="specaug",
+    classes=dict(
+        specaug=SpecAug,
+    ),
+    type_check=AbsSpecAug,
+    default=None,
+    optional=True,
+)
```

Comment:
When will `specaug` be used?

Reply:
[Here](https://github.com/espnet/espnet/pull/5181/files#diff-5f570b15a65d98e3e8992f846a74405495a848613069e73df8ecdf8157d1cad8R216)

Reply:
If the input is already a discrete sequence, why would you still need the SpecAugment?

Reply:
We verified that the time-masking is helping. Frequency-masking probably does not make sense. But the time-warping needs to be further verified. And the specaug is after the embedding layer.